inputPage.php

```diff
@@ -0,0 +1,62 @@
+<?php
+require_once('functions.php');
+$db = linkDB();
```

Comment:
rather than querying the whole db, it might be more efficient to get the input and then search the db for any matching entries.

Edit: I'm guessing these lines will either be shifted to a different file or be expanded on in the next few tasks

Reply:
yeah they probably will haha. But i think i'll still probably be selecting the DB with all the relevant fields even if some rows are missing values in fields